fri/server/main.py

```diff
@@ -47,6 +48,59 @@ def upload(dir):
         resp.status_code = 500
         return resp
 
+@app.route('/build/<dir>', methods=['POST'])

```

Comment:
dir should be the whole input as in "demo/sample" rather than hard-coding "demo" below.